pwa/src/styles/_setting.scss

```diff
@@ -0,0 +1,7 @@
+//colors
+
+$fresh: #849974;
```

Comment:
Please rename this file to be `colors.scss` with `$white: #ffffff` and start using this `NavBar.scss` and `Footer.scss`